server/banlist.js

```diff
@@ -0,0 +1,41 @@
+const crypto = require( "crypto" )
+const path = require( "path" )
+const fs = require('fs')
+let banlistHash = ''
+
+
+function getHash(){
+        if (banlistHash == '') {
+            const fileBuffer = fs.readFileSync('remote_banlist.txt')
+	        const hashSum = crypto.createHash('sha256')
+	        hashSum.update(fileBuffer)
+	        banlistHash = hashSum.digest('hex')
+        }
+        return banlistHash
+    }
+
+
+fs.watch('remote_banlist.txt', (curr, prev) => {
+    console.log('banlist changed')
+    const fileBuffer = fs.readFileSync('remote_banlist.txt')
+	const hashSum = crypto.createHash('sha256')
+	hashSum.update(fileBuffer)
+	banlistHash = hashSum.digest('hex')
+})
+
+
+module.exports = ( fastify, opts, done ) => {
+	
+
+    fastify.get( '/server/update_banlist', async ( request, response ) => {
```

Comment:
Might be worth changing to
```suggestion
    fastify.get( '/server/banlist_hash', async ( request, response ) => {
```
for easy of understanding